documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1360,12 +1418,14 @@ This property contains a comma-separated list of fully-qualified tables _(SCHEMA
 changes to the structure of the `source` block in order to unify the exposed structure across
 all the connectors. +
 By setting this option to `v1` the structure used in earlier versions can be produced.
-Note that this setting is not recommended and is planned for removal in a future Debezium version.
+Note that this setting is not recommended and is planned for removal in a future {ProductName} version.
 
 |`sanitize.field.names`
```

Comment:
Same as mentioned above, if Avro isn't something intended to be supported or called out, should this config option simply be omitted from downstream docs since its default suffices?

Reply:
See above. We wouldn't have our own Avro converters, but usage together with the ones from Apicurio will be supported.